languages/csharp/exercises/concept/inheritance/.meta/design.md

```diff
@@ -0,0 +1,48 @@
+# Design
+
+## Goal
+
+The goal of this exercise is to teach the student the Concept of Inheritance in C#.
+
+## Learning objectives
+
+- Know what inheritance is.
+- Know how to inherit from a class.
+- Know that all types inherit from `object`.
```

Comment:
I know this is only draft, but best to comment early ;)
Are abstract, sealed, virtual and protected, separate concepts to "basic" inheritance?
Seems like this exercise might be trying to cover too many aspects of inheritance with these all in the one exercise.

Reply:
Could be. @iHiD do you think it is teaching perhaps too much and that is why we were having the amount of text discussion?

Reply:
Maybe. Maybe it's split so virtual/abstract/etc go into this exercise, then protected/private/etc go into another? 

I actually don't feel there's too much here, I just think the examples are too separated and could be merged. But I've been doing OOP forever so I'm totally biased that this isn't too hard. It might be solid for someone new to OOP and two exercises might make more sense.

> do you think it is teaching perhaps too much and that is why we were having the amount of text discussion?

I think that's generally a good rule of thumb probably.

Reply:
@wolf99 I've cut down on a lot of the text.